action/setup.py

```diff
@@ -47,7 +48,7 @@
     entry_points={  # Optional
         'console_scripts': [
             'usgs-boundary=usgs_boundary.command:main',
-            'usgs-stac=usgs_boundary.stac:main',
+            # 'usgs-stac=usgs_boundary.stac:main',
```

Comment:
why was this commented out? The CI calls this to generate the tree.

Reply:
I don't see that file or it being called anywhere.
```
(usgs-lidar-update) ~/code/usgs-lidar/action % usgs-stac
Traceback (most recent call last):
  File "/Users/kmann/miniconda3/envs/usgs-lidar-update/bin/usgs-stac", line 33, in <module>
    sys.exit(load_entry_point('usgs-boundary', 'console_scripts', 'usgs-stac')())
  File "/Users/kmann/miniconda3/envs/usgs-lidar-update/bin/usgs-stac", line 25, in importlib_load_entry_point
    return next(matches).load()
  File "/Users/kmann/miniconda3/envs/usgs-lidar-update/lib/python3.10/importlib/metadata/__init__.py", line 171, in load
    module = import_module(match.group('module'))
  File "/Users/kmann/miniconda3/envs/usgs-lidar-update/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1004, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'usgs_boundary.stac'
```

Reply:
As far as I can see, the boundaries.yml is only calling usgs-boundary as a command

Reply:
ah, I never wired that up. nvm.